diode/diode_test.go

```diff
@@ -11,17 +12,20 @@ import (
 	diodes "code.cloudfoundry.org/go-diodes"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/diode"
+	"github.com/rs/zerolog/internal/cbor"
 )
 
 func ExampleNewWriter() {
 	d := diodes.NewManyToOne(1000, diodes.AlertFunc(func(missed int) {
 		fmt.Printf("Dropped %d messages\n", missed)
 	}))
-	w := diode.NewWriter(os.Stdout, d, 10*time.Millisecond)
+	buf := bytes.Buffer{}
+	w := diode.NewWriter(&buf, d, 10*time.Millisecond)
 	log := zerolog.New(w)
 	log.Print("test")
 
 	w.Close()
+	fmt.Println(cbor.DecodeIfBinaryToString(buf.Bytes()))
```

Comment:
Debug left-over?

Reply:
I have three options with tests:
1. Disable them for binary (since the tests compare the output with text output).
2. Decode if binary and let the comparison with text output continue..
3. Go with Option 1 - but create another file which does the same test for binary

I chose 2nd option.

Reply:
A cleaner approach occurred to me:
Since examples show up in documentation, i made it just text - so doc looks clean
Same examples, i made sure that there were tests that tested both ascii and binary - so i split diode_test.go to diode_example_test.go (text only) / diode_test.go (text and binary).